upgrade-toolkit/CDH-Discovery-Tool/mac-discovery-bundle-builder/config/log-config.yaml

```diff
@@ -0,0 +1,26 @@
+version: 1
+disable_existing_loggers: true
+formatters:
+  simple:
+    format: '%(levelname)s - %(threadName)s - %(module)s - %(asctime)s - %(message)s'
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: DEBUG
+    formatter: simple
+    stream: ext://sys.stdout
+  file:
+    class: logging.FileHandler
+    formatter: simple
+    level: DEBUG
+    filename: logs/mac-discovery-bundle-builder.log
```

Comment:
Presumably then we do not support this being executed on Windows?

Reply:
Tool should be executed from one of the cluster gateway nodes, as a result Windows is not an option.